### ISSUE TITLE
Not getting immutable vlues from formValueSelector or getFormValues

### ISSUE THREAD

**Issue by bobmonteverde** [erikras/redux-form#1695]
I thought I had redux-form setup to use immutable data structures... all of my imports of 'reduxForm' are from 'redux-form/immutable' (but I am importing all other functions from 'redux-form').  Am I supposed to import the other functions from "redux-form/immutable" ?

I'm selecting a Field array that I was hoping would be an immutable Iterable (either Map or List), as well as the entire form's values... both are coming back plain JS objects/arrays.

**Comment by erikras** [comment]
Yes, everything should be imported from `'redux-form/immutable'`.
